local-interchain/interchain/genesis.go

```diff
@@ -62,7 +63,7 @@ func SetupGenesisWallets(config *types.Config, chains []ibc.Chain) map[ibc.Chain
 			for _, coin := range amount {
 				additionalWallets[chainObj] = append(additionalWallets[chainObj], ibc.WalletAmount{
 					Address: acc.Address,
-					Amount:  coin.Amount.Int64(),
+					Amount:  math.NewInt(coin.Amount.Int64()),
```

Comment:
Ah nice. Yea this makes sense because this got merged first:

https://github.com/strangelove-ventures/interchaintest/pull/679

Really surprised CI didn't catch this??

Reply:
Oh one more note... around backporting.
That PR is not backported yet: https://github.com/strangelove-ventures/interchaintest/pull/685

Reply:
Oh good catch. We should get that back ported. I can look at it.